repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/list-item.ftl

```diff
@@ -203,7 +203,7 @@
 
         <#if ['Bulletin', 'CaseStudy', 'SearchBank', 'Event']?seq_contains(item.class.simpleName) || (pageURL != pageNotFoundURL || ('uk.nhs.hee.web.beans.Guidance' == item.getClass().getName() && miniHubGuidancePathToURLMap[item.path]??))>
```

Comment:
The URL for `Bulletin [probably an existing bug], CaseStudy, SearchBank & Event` ends up with 404. Could we fix them ?

Reply:
Thanks for spotting this. The links are now resolving to the relevant urls/binaries. 

I've not displayed exactly as the listing pages for each doc type, something I thought may be worth discussing with design team before we refactor the entire code pice for this as we probably can create macros to be shared with the listItem code blocks above.

Reply:
Thanks @Raq-HEE - Moved `<#case "uk.nhs.hee.web.beans.Guidance">` case on `list-item.ftl` right above `default` case so as to ensure rendering non-minihub guidance pages (using `default` case). Have done this work as part of 29d75f73c74148f08988df7a3638153f96ca1aa9 commit. Could you take a look ? Thanks!